model/eventGroupModel.js

```diff
@@ -5,6 +5,10 @@ const eventGroupSchema = Mongoose.Schema({
         type: Mongoose.Types.ObjectId,
         required: true
     },
+    companyID: {
+        type: Mongoose.Types.ObjectId,
```

Comment:
Something to discuss for validation purposes. @srkacha @zirlman Since now anytime we are creating something that references another document from another collection, we check manually if that document exists. We should do research if there is a way that `mongoose` does that automatically/internally. Would make writing code a fair bit easier. Maybe create a Trello task for this.